### PR TITLE
BaiTap03-ThanhNX

### PR DESCRIPTION
**Pull request by at-thanhnguyen4**

### CODE REVIEW

File: BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai1ViewController.swift [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        let frame = CGRect(x: 50, y: 100, width: 100, height: 100)
+        let userAvatar = UIImageView(image: UIImage(named: "avatar"))
+        userAvatar.frame = frame
+        userAvatar.contentMode = .scaleToFill
+        view.addSubview(userAvatar)
+        
+        let userName = UILabel(frame: CGRect(x: 50, y: 200, width: 100, height: 30))
+        userName.text = "Name1"
+        userName.textAlignment = NSTextAlignment.center
+        userName.backgroundColor = UIColor(red: 116/255.0, green: 185/255.0, blue: 255/255.0, alpha: 1)
```
Comment (at-bienle) [inline BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai1ViewController.swift]: Please add space after and before `/`

File: BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai1ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+        let frame = CGRect(x: 50, y: 100, width: 100, height: 100)
+        let userAvatar = UIImageView(image: UIImage(named: "avatar"))
+        userAvatar.frame = frame
+        userAvatar.contentMode = .scaleToFill
+        view.addSubview(userAvatar)
+        
+        let userName = UILabel(frame: CGRect(x: 50, y: 200, width: 100, height: 30))
+        userName.text = "Name1"
+        userName.textAlignment = NSTextAlignment.center
```
Comment (at-bienle) [inline BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai1ViewController.swift]: No need `NSTextAlignment`

File: BaiTap03/ThanhNX/BaiTapView/BaiTapView/ViewController.swift [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+
```
Comment (at-bienle) [inline BaiTap03/ThanhNX/BaiTapView/BaiTapView/ViewController.swift]: Remove lines spacing

File: BaiTap03/ThanhNX/BaiTapView/BaiTapView/ViewController.swift [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+//  Copyright © 2019 PCI0008. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
```
Comment (at-bienle) [inline BaiTap03/ThanhNX/BaiTapView/BaiTapView/ViewController.swift]: Remove unused code

File: BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai1ViewController.swift (hunk 2 above)
Comment (at-thanhnguyen4) [reply BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai1ViewController.swift]: i fixed it. Thank you

File: BaiTap03/ThanhNX/BaiTapView/BaiTapView/ViewController.swift (hunk 3 above)
Comment (at-thanhnguyen4) [reply BaiTap03/ThanhNX/BaiTapView/BaiTapView/ViewController.swift]: yes, i did

File: BaiTap03/ThanhNX/BaiTapView/BaiTapView/ViewController.swift (hunk 4 above)
Comment (at-thanhnguyen4) [reply BaiTap03/ThanhNX/BaiTapView/BaiTapView/ViewController.swift]: yes, i did

File: BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai2ViewController.swift [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+        User(name: "Thoa", avatar: #imageLiteral(resourceName: "img7")),
+        User(name: "Thao", avatar: #imageLiteral(resourceName: "img6")),
+        User(name: "Thuy", avatar: #imageLiteral(resourceName: "img9")),
+        User(name: "Thuong", avatar: #imageLiteral(resourceName: "img2")),
+        User(name: "Thinh", avatar: #imageLiteral(resourceName: "img5")),
+        User(name: "Thu", avatar: #imageLiteral(resourceName: "img3")),
+        User(name: "Thong", avatar: #imageLiteral(resourceName: "img8"))]
+    
+    override func viewDidLoad() {
+        struct Config {
```
Comment (at-bienle) [inline BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai2ViewController.swift]: Please move struct out of function

File: BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai2ViewController.swift [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+            static let nameWidth: CGFloat = 100
+            static let nameHeight: CGFloat = 130
+        }
+        super.viewDidLoad()
+        func prints() {
```
Comment (at-bienle) [inline BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai2ViewController.swift]: Why you define a function in function?

File: BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai2ViewController.swift [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+        
+        let userAvatarImageView = UIImageView(image: avatarImage)
+        userAvatarImageView.frame = CGRect(x: 0, y: 0, width: 90, height: 100)
+        userAvatarImageView.contentMode = .scaleToFill
+        containerView.addSubview(userAvatarImageView)
+        
+        let userNameLabel = UILabel(frame: CGRect(x: 0, y: 100, width: 90, height: 30))
+        userNameLabel.text = name
+        userNameLabel.textAlignment = .center
+        userNameLabel.backgroundColor = UIColor(red: 116/255, green: 185/255, blue: 255/255, alpha: 1)
```
Comment (at-bienle) [inline BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai2ViewController.swift]: Check convention here

File: BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai3ViewController.swift [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+//  BaiTapView
+//
+//  Created by PCI0008 on 6/19/19.
+//  Copyright © 2019 PCI0008. All rights reserved.
+//
+
+import UIKit
+
+class Bai3ViewController: UIViewController {
+    let x = (UIScreen.main.bounds.width / 2) - (Config.sliderWidth / 2)
```
Comment (at-bienle) [inline BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai3ViewController.swift]: No need `()`

File: BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai3ViewController.swift [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+    let y = (UIScreen.main.bounds.height / 2) - (Config.sliderHeight / 2)
```
Comment (at-bienle) [inline BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai3ViewController.swift]: No need `()`

File: BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai4ViewController.swift [hunk 10]
```diff
@@ -0,0 +1,112 @@
...
+                             Calculator(cal: "7"),
+                             Calculator(cal: "8"),
+                             Calculator(cal: "9"),
+                             Calculator(cal: "x"),
+                             Calculator(cal: "AC"),
+                             Calculator(cal: "0"),
+                             Calculator(cal: "="),
+                             Calculator(cal: "/")]
+
+    override func viewDidLoad() {
```
Comment (at-bienle) [inline BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai4ViewController.swift]: Please check your code here!

File: BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai2ViewController.swift (hunk 5 above)
Comment (at-thanhnguyen4) [reply BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai2ViewController.swift]: Yes. I has moved. Thank you

File: BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai3ViewController.swift (hunk 9 above)
Comment (at-thanhnguyen4) [reply BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai3ViewController.swift]: Yes. I fixed it. Thank you

File: BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai3ViewController.swift (hunk 8 above)
Comment (at-thanhnguyen4) [reply BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai3ViewController.swift]: Yes. I fixed it. Thank you

File: BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai4ViewController.swift (hunk 10 above)
Comment (at-thanhnguyen4) [reply BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai4ViewController.swift]: yes. i had checked. i has move struct out of func.

File: BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai2ViewController.swift (hunk 7 above)
Comment (at-thanhnguyen4) [reply BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai2ViewController.swift]: yes. i fixed it. thank you

File: BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai2ViewController.swift (hunk 6 above)
Comment (at-thanhnguyen4) [reply BaiTap03/ThanhNX/BaiTapView/BaiTapView/Bai2ViewController.swift]: i had remove this function. thank you.

**Pull request merged**